my-tweet/src/context/post-context.js

```diff
@@ -8,8 +8,10 @@ const PostProvider=({children})=>{
     const [newPostData,setNewPostData]=useState()
     const [login,setLogin]=useState({username:"",password:""})
```

Comment:
If you use more than three ` useState ` then you should use useReducer hook so your code looks more cleaner and readable

Reply:
ok will do it